test/parallel/test-module-loading-error.js

```diff
@@ -74,7 +74,7 @@ const re = /^The "id" argument must be of type string\. Received type \w+$/;
 common.expectsError(
   () => { require(''); },
   {
-    type: Error,
+    type: RangeError,
```

Comment:
Without changing the errors that result in such changes, the PR would be a patch. I personally prefer a PR that is possible to backport instead of having the PR as semver-major because just very few cases that were wrong before.

Reply:
I can remove the semver-major changes, but I think a backport would have to be done from scratch anyway, even for v9.x, because many error codes don't exist in any release line

Reply:
Most errors were already ported in 9.x as far as I know. I personally still think it would be better to have this as a semver-patch but I am fine with it if others want to land it as is.